### PR TITLE
Fixed a 'major' lack of functionality in user task submission - hope …

### PR DESCRIPTION
**Pull request by damm89**
…this works now.

### CODE REVIEW

File: backend/src/zac/camunda/api/serializers.py [hunk 1]
```diff
@@ -39,12 +40,18 @@ class ProcessInstanceSerializer(serializers.Serializer):
     tasks = TaskSerializer(many=True)
 
 
+class ChoiceFieldNoValidation(serializers.ChoiceField):
+    def to_internal_value(self, data):
+        if not isinstance(data, str):
+            raise serializers.ValidationError("Form needs to be a string.")
+
```
Comment (sergei-maertens) [inline backend/src/zac/camunda/api/serializers.py:47]: Shouldn't this return `data`?

File: backend/src/zac/camunda/api/serializers.py [hunk 2]
```diff
@@ -76,28 +83,37 @@ def __init__(self, *args, **kwargs):
...
+        lookup = (
+            form_key
+            if form_key in self.serializer_mapping
+            else self.fallback_distriminator_value
+        )
+        return self.serializer_mapping[lookup]
+
+    def is_valid(self, raise_exception=True):
+        super().is_valid(raise_exception=raise_exception)
+        self.get_mapped_serializer().is_valid(raise_exception=raise_exception)
```
Comment (sergei-maertens) [inline backend/src/zac/camunda/api/serializers.py:103]: I'd expect this to be the job of the `PolymorphicSerializer` base class, no?

**Review by sergei-maertens — CHANGES_REQUESTED** [review]
Looks okay, but I'm not sure if this is a problem in the base class or this specific user task serializer?

I'd also like to see some regression tests that fail without this patch and pass with the patch.

**Comment by damm89** [comment]
> Looks okay, but I'm not sure if this is a problem in the base class or this specific user task serializer?
> 
> I'd also like to see some regression tests that fail without this patch and pass with the patch.

Basically the only problem was that the ChoiceField for the form key was throwing validation errors...

**Review by sergei-maertens — APPROVED** [review]

**Comment by sergei-maertens** [comment]
I'd still expect a regression test though!

**Review by sergei-maertens — APPROVED** [review]

**Pull request merged**